docs/src/Sheet.doc.js

```diff
@@ -23,472 +30,445 @@ card(
         required: false,
         defaultValue: null,
-        description: [
-          'Supply the container element or render prop that is going to be used as the Sheet main content.',
-          'When using a render prop, just pass the argument onDismissStart to your exitpoint action elements.',
-          'Obs: This element will be padded by 32px, differently than `<Modal>`.',
-        ],
-        href: 'defaultPaddingAndStylingExample',
+        description:
+          "Supply the container element(s) or render prop that will be used as the Sheet's main content. See the [animation variant](#Animation) for info on how to add exit animations to Sheet content.",
       },
       {
         name: 'closeOnOutsideClick',
         type: 'boolean',
         required: false,
         defaultValue: true,
-        description: [
-          'Indicate whether clicking on the backdrop (gray area) outside of the Sheet will automatically close it.',
-          'Possible values:',
-          '- true: clicking on the backdrop will close the Sheet.',
-          '- false: clicking on the backdrop will keep the Sheet open.',
-        ],
-        href: 'closeOnOutsideClickExample',
+        description:
+          'Indicate whether clicking on the backdrop (gray area) outside of the Sheet will automatically close it. See the [outside click variant](#Preventing-close-on-outside-click) for more info.',
       },
       {
         name: 'footer',
         type: 'React.Node | (({| onDismissStart: () => void |}) => React.Node)',
         required: false,
         defaultValue: null,
-        description: [
-          'Supply the container element or render prop that is going to be used as the Sheet custom footer.',
-          'When using a render prop, just pass the argument onDismissStart to your exitpoint action elements.',
-          'Obs: This element will be padded by 32px, similarly to `<Modal>`.',
-        ],
-        href: 'defaultPaddingAndStylingExample',
+        description:
+          "Supply the container element(s) or render prop that will be used as the Sheet's custom footer. See the [footer variant](#Footer) for more info.",
       },
       {
         name: 'heading',
         type: `string`,
         required: false,
         defaultValue: null,
-        description: [
-          'Supply the text that is going to be placed as the Sheet text heading.',
-          'Please do not repeat the same text being passed in the "accessibilitySheetLabel" prop, but instead provide something that identifies the Sheet. For instance, if the heading is "Pin Builder", the accessibilitySheetLabel can be "Create a new Pin".',
-          'Obs: This element will be padded by 32px, similarly to `<Modal>`.',
-        ],
-        href: 'defaultPaddingAndStylingExample',
+        description:
+          "Supply the text that is going to be placed as the Sheet's text heading. Be sure to localize this text. See the [heading variant](#Heading) for more info.",
```

Comment:
"is going to be placed as" is a bit wordy…perhaps something like "Text used for Sheet's heading."?

Also some articles that could be removed here and throughout, but that can always be follow-up work and I won't call them out here 🙂